backend.config.js

```diff
@@ -1,20 +1,44 @@
+// Also wss://kusama-rpc.polkadot.io
+const DEFAULT_WS_PROVIDER_URL = 'wss://substrate-node:9944';
+
 module.exports = {
-  // Local Polkadot Kusama node
-  wsProviderUrl: 'ws://substrate-node:9944',
-  // Postgres database connection params
   postgresConnParams: {
     user: 'polkastats',
     host: 'postgres',
     database: 'polkastats',
     password: 'polkastats',
     port: 5432,
   },
-  BLOCK_HARVESTER_POLLING_TIME: 1 * 60 * 1000, // Run every 10 min
-  ACTIVE_ACCOUNTS_POLLING_TIME: 1 * 60 * 1000, // Run every 60 min
-  PHRAGMEN_POLLING_TIME: 5 * 60 * 1000, // Run every 5 min
+
+  crawlerRunner: {
+    wsProviderUrl: process.env.WS_PROVIDER_URL || DEFAULT_WS_PROVIDER_URL,
```

Comment:
Why we need to include `wsProviderUrl` in all crawlers?

I would use:

```
// Also wss://kusama-rpc.polkadot.io
const DEFAULT_WS_PROVIDER_URL = 'wss://substrate-node:9944';

module.exports = {
  postgresConnParams: {
    user: 'polkastats',
    host: 'postgres',
    database: 'polkastats',
    password: 'polkastats',
    port: 5432,
  },
  wsProviderUrl: process.env.WS_PROVIDER_URL || DEFAULT_WS_PROVIDER_URL,
  crawlers: {
    blockListener: {
      enabled: true,
      startingDelay: false
    },
    blockHarvester: {
      enabled: true,
      startingDelay: true,
      pollingTime: 1 * 60 * 1000,
    },
    staking: {
      enabled: true,
      startingDelay: false
    },
    rewards: {
      enabled: true,
      startingDelay: false
    },
    accounts: {
      enabled: true,
      startingDelay: true,
      pollingTime: 1 * 60 * 1000,
    },
    phragmen: {
      enabled: true,
      startingDelay: true,
      pollingTime: 5 * 60 * 1000,
      phragmenOutputDir: '/tmp/phragmen',
      offlinePhragmenPath: '/usr/app/polkastats-backend-v3/offline-phragmen',
    }
  }
};
```

Reply:
Done

Reply:
dont see yet!

Reply:
https://github.com/Colm3na/polkastats-backend-v3/pull/69/files#diff-727a97cd278f2750a0af301c4beb6f6a

Reply:
But we can store all the crawlers in an array and loop over it as you suggested ^^

Reply:
Done